apps/s_server.c

```diff
@@ -751,23 +762,25 @@ static int get_ocsp_resp_from_responder(SSL *s, tlsextstatusctx *srctx,
                                         STACK_OF(OCSP_RESPONSE) **sk_resp)
 {
     X509 *ssl_cert = NULL;
+    SSL_CTX *ssl_ctx;
     int i, num = 0;
-    STACK_OF(X509) *server_certs = NULL;
+    STACK_OF(X509) *server_chain = NULL;
     OCSP_RESPONSE *resp = NULL;
 
     if (*sk_resp != NULL) {
         sk_OCSP_RESPONSE_pop_free(*sk_resp, OCSP_RESPONSE_free);
         *sk_resp = NULL;
     }
 
-    SSL_get0_chain_certs(s, &server_certs);
+    ssl_ctx = SSL_get_SSL_CTX(s);
+    SSL_CTX_get0_chain_certs(ssl_ctx, &server_chain);
```

Comment:
And why is it necessary to change from SSL_get0_chain_certs() to SSL_CTX_get0_chain_certs()?